tpcx_bb/queries/q04/tpcx_bb_query_04.py

```diff
@@ -51,11 +57,18 @@ def read_tables():
 
 
 def abandonedShoppingCarts(df, DYNAMIC_CAT_CODE, ORDER_CAT_CODE):
+    import cudf
+
+    # work around for https://github.com/rapidsai/cudf/issues/5470
+    df.reset_index(drop=True, inplace=True)
```

Comment:
Since 5470 is fixed, we can get rid of this comment (potentially get rid of thee reset_index calls as well if possible).

Reply:
Agreed. In the interest of time, let's replace this with a TODO for now to test removing the reset index calls

Reply:
Done